doc-content/enterprise-only/weblogic/wls-install-start-proc.adoc

```diff
@@ -12,7 +12,7 @@ If you are already running an instance of {WEBLOGIC} that uses the same listener
 
 .Procedure
 . Download {WEBLOGIC} 12.2.1.3.0 or later from the {WEBLOGIC} http://www.oracle.com/technetwork/middleware/weblogic/downloads/index.html[Downloads page].
-. Sign in to the target system and verify that a certified JDK already exists on your system. The installer requires a certified JDK. For system requirements, see http://www.oracle.com/technetwork/middleware/ias/downloads/fusion-requirements-100147.html[Oracle Fusion Middleware Systems Requirements and Specifications]. To download the JDK, see https://docs.oracle.com/middleware/12213/lcm/ASINS/GUID-A3738BBA-5FD1-425B-91EB-7CB22187C700.htm#ASINS355[About JDK Requirements for an Oracle Fusion Middleware Installation].
+. Sign in to the target system and verify that a certified JDK already exists on your system. The installer requires a certified JDK. For system requirements, see http://www.oracle.com/technetwork/middleware/ias/downloads/fusion-requirements-100147.html[Oracle Fusion Middleware Systems Requirements and Specifications]. To download the JDK, see the 'About JDK Requirements for an Oracle Fusion Middleware Installation' section in the  https://docs.oracle.com/en/middleware/lifecycle/12.2.1.3/asins/preparing-oracle-fusion-middleware-installation.html#GUID-A3738BBA-5FD1-425B-91EB-7CB22187C700[Planning an Installation of Oracle Fusion Middleware].
```

Comment:
Suggestion - the linked title name appears to be different:
s/[Planning an Installation of Oracle Fusion Middleware]/[Preparing for an Oracle Fusion Middleware Installation]/

Minor suggestion - "the" could be removed for the second link and the step can be broken into two paragraphs:

```
. Sign in to the target system and verify that a certified JDK already exists on your system. The installer requires a certified JDK.
+
For information about system requirements, see http://www.oracle.com/technetwork/middleware/ias/downloads/fusion-requirements-100147.html[Oracle Fusion Middleware System Requirements and Specifications].
+
To download the JDK, see the 'About JDK Requirements for an Oracle Fusion Middleware Installation' section in  https://docs.oracle.com/en/middleware/lifecycle/12.2.1.3/asins/preparing-oracle-fusion-middleware-installation.html#GUID-A3738BBA-5FD1-425B-91EB-7CB22187C700[Preparing for an Oracle Fusion Middleware Installation].
```

Reply:
Thanks, @csherrar for the suggestion.